storage-proofs/post/src/fallback/vanilla.rs

```diff
@@ -364,10 +365,9 @@ impl<'a, Tree: 'a + MerkleTreeTrait> ProofScheme<'a> for FallbackPoSt<'a, Tree>
             num_sectors_per_chunk,
         );
 
-        let mut partition_proofs = Vec::new();
-
         // Use `BTreeSet` so failure result will be canonically ordered (sorted).
         let mut faulty_sectors = BTreeSet::new();
+        let mut partition_proofs = Vec::new();
```

Comment:
Similar comment to above — is there a significance to this reordering?

Reply:
I'll look at this one -- it's likely a rebase artifact.

Reply:
Updated